homeassistant/components/recollect_waste/sensor.py

```diff
@@ -87,6 +93,11 @@ def __init__(self, coordinator: DataUpdateCoordinator, entry: ConfigEntry) -> No
         self._entry = entry
         self._state = None
 
+    @property
+    def device_class(self) -> dict:
+        """Return the device class."""
+        return DEVICE_CLASS_TIMESTAMP
```

Comment:
We don't set a custom icon when setting device class.

Reply:
Sorry @MartinHjelmare, unclear on what the ask is?

Reply:
We should remove the `icon` property. The frontend should set a specific icon per device class.